README.md

```diff
@@ -5,10 +5,10 @@ ContinuedFractions.jl
 ```julia
 julia> using ContinuedFractions
 
-julia> cf = ContinuedFraction(sqrt(2))
+julia> cf = ContinuedFraction(sqrt(big(2)))
```

Comment:
You're right. I didn't think it through. Also the next line should be changed. Maybe it shouldn't be a BigInt in the example after all. I was using the package to solve diophantine equations and was surprised that it failed on 61. But maybe it's up to the Julia user to consider this. What do you think?